pkg/env/web_env.go

```diff
@@ -163,7 +163,7 @@ func Environ() []string {
 // to fetch ENV values for the env value from a remote server.
 func LookupEnv(key string) (string, bool) {
 	v, ok := os.LookupEnv(key)
-	if ok && strings.HasPrefix(v, webEnvScheme) {
+	if ok && key == "MINIO_ARGS" {
```

Comment:
Lookup ENV is generic is this for testing?

Reply:
Lookup ENV is not generic, it assumes a GET is needed on every env variable that starts with `env` which is not the case

Reply:
it is the case it is a generic implementation @nitisht the webhook server needs to implement the relevant part i.e to provide the expected ENV 

This change introduces bugs, this is not how MinIO side of the code should behave.